README.md

```diff
@@ -45,7 +45,8 @@ Details for setting up the IDE is described in the wiki (https://github.com/cgeo
 
 ### API keys
 
-Google doesn't hand out new keys for Google Maps v1, you have to use an existing one. You can leave it empty, but then Google Maps doesn't work. 
+Google doesn't hand out new keys for Google Maps v1, you have to use an existing one. You can leave it empty, but then Google Maps doesn't work.
+Also to be able to use Google Maps you need to use a Google Api enabled image, so make sure to select the right image for your emulator / mobile device, otherwise Google Maps won't be offered as map provider in c:geo. 
```

Comment:
https://en.wikipedia.org/wiki/Slash_(punctuation)#Spacing
Since you use "mobile device" here, you may even get away with the spaces, but I'd prefer "emulator/device".

Reply:
done